setup.py

```diff
@@ -37,6 +37,7 @@ def get_advanced_templates():
         'aiohttp==0.22.5',
         'analytics-python',
         'coloredlogs',
+        'dnspython',
```

Comment:
@mellenburg here we go: [team-city-link](https://teamcity.mesosphere.io/viewLog.html?buildId=505941&buildTypeId=ClosedSource_Dcos_IntegrationTests_CloudIntegrationTests_DcosOssUpgradeTestOnPre) 

* One way would be introduce a tox target and invoke `test-upgrade-vpc` from the tox target.